extensions/wikia/PortableInfobox/templates/PortableInfoboxItemImage.mustache

```diff
@@ -1,7 +1,15 @@
 <div class="portable-infobox-item item-type-image no-margins">
 	<figure class="portable-infobox-image-wrapper">
-		<a href="{{url}}" class="image image-thumbnail" title="{{alt}}">
-			<img src="{{thumbnail}}" class="portable-infobox-image" alt="{{alt}}" width="{{{width}}}" height="{{{height}}}" data-image-key="{{name}}" data-image-name="{{name}}"/>
+		<a href="{{url}}" class="image image-thumbnail{{#isVideo}} video video-thumbnail small{{/isVideo}}"
```

Comment:
it doesn't change anything in the functionality except that it will open full size image if user has JS turned off